.github/workflows/VercelDeploy.yaml

```
@@ -37,19 +40,34 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
+      - name: Run Keystone dev with timeout
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2016:info:1:19: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)